addons/spreadsheet_account/static/tests/ui/accounting_drilldown.test.js

```diff
@@ -51,6 +51,7 @@ test("Create drill down domain", async () => {
                             range_type: "year",
                             year: 2020,
                         },
+                        partner_ids: [""],
```

Comment:
Nothing needs to change here right?

Wouldn't it be an issue for the stable if it changed?